diff --git a/node_modules/react-native-permissions/ios/RNPermissions.m b/node_modules/react-native-permissions/ios/RNPermissions.m
index ff7f80e..21d979b 100644
--- a/node_modules/react-native-permissions/ios/RNPermissions.m
+++ b/node_modules/react-native-permissions/ios/RNPermissions.m
@@ -207,7 +207,7 @@ - (NSDictionary *)constantsToExport {
     [message appendString:@"• Uninstall this app, delete your Xcode DerivedData folder and rebuild it.\n"];
     [message appendString:@"• If you use `use_frameworks!`, follow the workaround guide in the project README."];
 
-    RCTLogError(@"%@", message);
+    // RCTLogError(@"%@", message);
   }
 #endif
 
